Modules/_zstd/_zstdmodule.c

```diff
@@ -7,7 +7,13 @@ Python module.
 #  define Py_BUILD_CORE_MODULE 1
 #endif
 
+#include "Python.h"
```

Comment:
I never remember whether we explicitly include Python.h or not, considering `_zstdmodule.h` is already including it.

Reply:
Me neither, I'll check to see what's most common

Reply:
Me neither, I'll check to see what's most common

Reply:
I've removed them from the headers